code/game/verbs/who.dm

```diff
@@ -179,14 +182,14 @@
 			if(CLIENT_IS_STAFF(src))
 				if(entry.admin_holder?.fakekey)
 					dat += " <i>(HIDDEN)</i>"
-			if(istype(entry.mob, /mob/dead/observer))
-				dat += "<B> - </B><B style='color:#777'>Observing</B>"
-			else if(istype(entry.mob, /mob/new_player))
-				dat += "<B> - </B><font color='#000'>Lobby</font></B>"
-			else
-				dat += "<B> - </B><B style='color:#688944'>Playing</B>"
-			if(entry.is_afk())
-				dat += "<B style='color:#4D0096'> (AFK)</B>"
+				if(istype(entry.mob, /mob/dead/observer))
+					dat += "<B> - </B><B style='color:#777'>Observing</B>"
+				else if(istype(entry.mob, /mob/new_player))
+					dat += "<B> - </B><font color='#000'>Lobby</font></B>"
```

Comment:
can you make all the ones which are pure black (like lobby, dead in `who`), show up in a colour that is not pure black? because they are very hard to read against the background